docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/Hash.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class Hash
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext(seed: 1);
+
+            // Get a small dataset as an IEnumerable.
+            var rawData = new[] {
+                new InputData() { Category = "MLB" , Age = 18 },
+                new InputData() { Category = "NFL" , Age = 14 },
+                new InputData() { Category = "NFL" , Age = 15 },
+                new InputData() { Category = "MLB" , Age = 18 },
+                new InputData() { Category = "MLS" , Age = 14 },
+            };
+
+            var data = mlContext.Data.LoadFromEnumerable(rawData);
+
+            // Construct the pipeline that would hash the two columns and store the results in new columns.
+            var pipeline = mlContext.Transforms.Conversion.Hash("CategoryHashed", "Category", numberOfBits: 16, maximumNumberOfInverts: 2)
+                          .Append(mlContext.Transforms.Conversion.Hash("AgeHashed", "Age", numberOfBits: 8));
+
+            // Let's fit our pipeline, and then apply it to the same data.
+            var transformer = pipeline.Fit(data);
+            var transformedData = transformer.Transform(data);
+
+            // Store the post transformation from the IDataView format, to an IEnumerable<TransformedData> for easy consumption
+            var convertedData = mlContext.Data.CreateEnumerable<TransformedData>(transformedData, true);
+            foreach (var item in convertedData)
+            {
+                Console.WriteLine($"Category: {item.Category} - CategoryHashed: {item.CategoryHashed}. Age: {item.Age} - AgeHashed {item.AgeHashed}");
+            }
+
+            // Output
```

Comment:
>Output [](start = 15, length = 6)

Expected? Do we have a standard we're using?

Reply:
Good point, we don't. we might want to do a pass over all of them and just nit at things like this one. 

---
In reply to: [267928171](https://github.com/dotnet/machinelearning/pull/3042#discussion_r267928171) [](ancestors = 267928171)